server/routes/api.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express')
+const router = express.Router()
+const axios = require('axios')
+
+const Transaction = require('../model/Transaction')
+
+router.get('/transactions', function(req, res){
+    Transaction.find({}, (err, data)=>res.send(data))
```

Comment:
use async await here if you use it in the rest of your app

`const transactions = await Transaction.find({})`
`res.send(transactions)`